recipe/meta.yaml

```diff
@@ -24,13 +24,13 @@ requirements:
     - setuptools
     - numpy
     - fftw >=3.3.8
-    - liblapack >=3.8
+    - liblapack >=3.8=*openblas
     - openblas
   run:
     - python
     - numpy
     - fftw >=3.3.8
-    - liblapack >=3.8
+    - liblapack >=3.8=*openblas
```

Comment:
Unless you can't build against the netlib version of lapack for some special reason, I don't think this is correct thing to do.